plugins/extractors/metabase/metabase.go

```diff
@@ -25,15 +25,17 @@ var summary string
 
 var sampleConfig = `
 host: http://localhost:3000
+instance_label: my-metabase
 user_id: meteor_tester
 password: meteor_pass_1234`
 
 // Config holds the set of configuration for the metabase extractor
 type Config struct {
-	Host      string `mapstructure:"host" validate:"required"`
-	Username  string `mapstructure:"username" validate:"required"`
-	Password  string `mapstructure:"password" validate:"required"`
-	SessionID string `mapstructure:"session_id"`
+	Host          string `mapstructure:"host" validate:"required"`
+	InstanceLabel string `mapstructure:"instance_label" validate:"required"`
+	Username      string `mapstructure:"username" validate:"required"`
+	Password      string `mapstructure:"password" validate:"required"`
+	SessionID     string `mapstructure:"session_id"`
```

Comment:
@StewartJingga with SessionID now in the config, the Username and Password are optional right?? So I think the `validate:"required"` should be removed.

Reply:
@GrayFlash updated 🙏